mlir/lib/Analysis/Presburger/Simplex.cpp

```diff
@@ -639,17 +649,18 @@ class GBRSimplex {
 };
 
 // Let b_{level}, b_{level + 1}, ... b_n be the current basis.
-// Let F_i(v) = max <v, x - y> where x and y are points in the original polytope
```

Comment:
Turns out that my clang-format vim binding doesn't actually change this anymore. Perhaps I accidentally run clang-format with a wrong config at some point. I will fix all the comment changes.